### ISSUE TITLE
Sensors and other cozylife utils

### ISSUE THREAD

**Issue by wikiyu** [cozylife/hass_cozylife_local_pull#36]
Will there be any chance of adding support for more cozylife items? I have some Temp&Humidity sensors that are not recognized by this addon.

**Comment by giaur500** [comment]
Nope. No chance. I'd suggest to throw away cozylife sensors and use Tuya instead
